instance/tests/decorators.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Decorators - Useful decorators for unit tests
+"""
+from unittest import mock
+
+
+def patch_git_checkout(func):
+    """
+    Patch git checkout process in order to allow mocking up the checkout
+    process and the working directory
+    """
+    def _wrap(*args, **kwargs):
```

Comment:
Could you use https://docs.python.org/3.4/library/functools.html#functools.wraps ?